### ISSUE TITLE
Bug: Fixing "Untextered" trait

### ISSUE THREAD

**Issue by tartakovsky** [lobster-dao/overview#43]
Just to document it here. In the middle of talking to rarity.tools they asked us to fix the Untextered trait before they'll list us. Klibansky is for fixing it too, as it said it wasn't intentional and is a typo. That would require uploading new metadata to ipfs and a tx changing the metadata link in the smart contract. Working on it.

**Comment by amacar** [comment]
Please fix it, it triggers my OCD. 😅

**Comment by RobAnon** [comment]
Untextured is not set as a "null" trait, and is included in the metadata. Fundamentally speaking, NFTs are meant to be immutable. I'm opposed to modifying any of my property without my express consent; I paid a fair amount of ETH for this NFT, and I am to have no say in how its value is decreased after-the-fact? 

The artist's intentions shouldn't matter if the art exists. The Leaning Tower of Pisa isn't famous because it's a well-built tower, it's famous because it is flawed. Leave the art as is.

**Comment by NerosBerg** [comment]
Agree with the above.

**Comment by tartakovsky** [comment]
just to be clear: nobody's removing any traits. There were two traits: Untextured and Untextered. The second one is the typo of the first one. After the change – both of them will be Untextured. The Untextured trait will stay in place.

**Comment by JonathanlDoe** [comment]
What's the difference between them? Apart from the typo... 😅

**Comment by zhongfu** [comment]
> What's the difference between them? Apart from the typo... 😅

@JonathanlDoe afaict there's no difference, it's just the typo

**Comment by kkkrackpot** [comment]
> What's the difference between them? Apart from the typo... 😅

Difference between what?
Some lobs have _Gene_ trait _UntextEted_ instead of _UntextUred_. It was a typo in the word.
The rest is the same as other lobs have.
Indeed, two _untextered_ (or 'normal' and _untextered_) may differ by other traits, images, etc.
I doubt there're two identical lobs at all, they're nfts, after all.

**Comment by tartakovsky** [comment]
Yeah, no difference, Klibansky just did a typo when he entered metadata. Both should be Untextured.

**Comment by JonathanlDoe** [comment]
So after the typo fix there will be only one Untextured trait, effectively  consolidating the two trait groups (untextUred and untextEred) under a single one - Unterxtured. Right?

**Comment by tartakovsky** [comment]
Yes.

**Comment by tartakovsky** [comment]
Just to follow the democratic process and to avoid future issues, here is the vote: https://snapshot.org/#/10b57e6da0.eth/proposal/0xda1aa1b11826bac7246cce93364a74bf7aa42502c0baf8ae060e995bf2b53288

Up for 3 days.

**Comment by JonathanlDoe** [comment]
Please don't hate me for this, but you do know that in numismatics or philately, the common items with production errors on them are a lot more valuable than the other common common items. Right? 😅

**Comment by tartakovsky** [comment]
Yeah, that has been voiced already. So far the consensus is that it's more important to remove the mistake, to both look normal, spare people the OCD triggers and proceed with Rarity listing, than to keep the mistake for that purpose. But I see that it's not 100% clear consensus, so there is the vote.

**Comment by JonathanlDoe** [comment]
Oh. Alright. Mr. Redundant here. Sorray! 🙏

**Comment by tartakovsky** [comment]
Not redundant, it's important that people say what they want publicly, especially here where it'll stay up longer than in the chat :)

**Comment by kkkrackpot** [comment]
Fixed.
Pls close.
